.github/workflows/build-test.yml

```
@@ -157,14 +157,21 @@ jobs:
           python -m pip install maturin
 
     - name: Build MacOs with maturin on Python ${{ matrix.python }}
-      if: matrix.os.matrix == 'macos'
+      if: matrix.os.matrix == 'macos' && matrix.arch.matrix == 'intel'
       env:
         MACOSX_DEPLOYMENT_TARGET: 10.14
```

How about this and avoid the duplication?  I mean, I hate this and/or form for picking values, but I'm getting used to it in CI.

```suggestion
        MACOSX_DEPLOYMENT_TARGET: ${{ matrix.arch.matrix == 'intel' && '10.14' || '' }}
```

---

Or maybe just always explicit?

```suggestion
        MACOSX_DEPLOYMENT_TARGET: ${{ matrix.arch.matrix == 'intel' && '10.14' || '11.0' }}
```

---

I'm not familiar with this syntax. It looks like it would all just evaluate to `True`. But I suppose these are bash-like operators.

---

it's working

---

Similar to Python, you don't get a boolean out of a boolean operator.  You get one of the operands.  `"" or [1, 2, 3]` in Python results in `[1, 2, 3]` because `""` is falsey and it's an `or` so it checks the second operand which is truthy and uses that as the resulting value.  It's like the questionability of ternary operators, but more so.  But, it's either this or some JSON string games and indexing...